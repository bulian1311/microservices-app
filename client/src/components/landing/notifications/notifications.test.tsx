import React from "react";
import { render, RenderResult } from "@testing-library/react";
import Notifications from "./notifications.component";

let documentBody: RenderResult;

beforeEach(() => {
  documentBody = render(<Notifications />);
});

test("Notifications component match snapshot.", () => {
  const { baseElement } = documentBody;
  expect(baseElement).toMatchSnapshot();
});
