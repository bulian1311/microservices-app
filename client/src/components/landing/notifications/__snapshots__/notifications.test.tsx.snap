// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Notifications component match snapshot. 1`] = `
<body>
  <div>
    <div
      class="sc-jSgupP dHXuMC"
    />
  </div>
</body>
`;
